calculator-project/src/Component/buttons.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react'
+import "../CSS/buttons.css"
+import "../App"
+
+
+class Buttons extends Component{
+
+   render(){
+
+       return(
+           <div className="buttons">
```

Comment:
This could have been a button class that loops through an array of elements and adds their value to each button